guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -373,24 +374,66 @@ These argument examples can be taken in the template like in the following examp
 
 /** @var string $someString */
 $someString = $block->getData('some_string');
+
 /** @var bool $isActive */
 $isActive = $block->getData('is_active');
+
 /** @var Vendor\CustomModule\ViewModel\Class|\Magento\Framework\View\Element\Block\ArgumentInterface $viewModel */
 $viewModel = $block->getData('viewModel');
+
 /** @var string|int|float $someNumber */
 $someNumber = $block->getData('some_number');
```

Comment:
Just a thought. Wouldn't it better, if the magic method examples were adjacent to their getData method examples? Might be more clear on the similarity and not partially duplicating a huge code block.

Reply:
Hi @drpayyne 

I did the adjustments combining the examples.
Thanks